Source/objects.cpp

```diff
@@ -4235,40 +4235,24 @@ void __fastcall OperateShrine(int pnum, int i, int sType)
 // 52571C: using guessed type int drawpanflag;
 // 676190: using guessed type int deltaload;
 
-void __fastcall OperateSkelBook(int pnum, int i, unsigned char sendmsg)
+void __fastcall OperateSkelBook(int pnum, int i, BOOL sendmsg)

```

Comment:
Don't forget to also update it in objects.h.

Reply:
:)